src/lib/rest/ConnectionInfo.h

```diff
@@ -147,18 +154,18 @@ class ConnectionInfo
   std::string                charset;
   std::string                tenantFromHttpHeader;
   std::string                tenant;
+  RestService*               restServiceP;
   std::vector<std::string>   servicePathV;
   HttpHeaders                httpHeaders;
   char*                      payload;
   int                        payloadSize;
-  char                       payloadWord[64];
   std::string                answer;
   int                        callNo;
   ParseData*                 parseDataP;
   unsigned short             port;
   std::string                ip;
   ApiVersion                 apiVersion;
-  RequestType                requestType;
+  RequestType                requestType;  // To Be Removed (found inside restServiceP->request (restServiceP->type))
```

Comment:
"To Be Removed" means something to be done in this PR or a mark for the future?

In the second case a "FIXME" mark should be a good addition as prefix to the comment message itself.

Reply:
Yeah, FIXME better than the current comment.
And no, not in this PR. It's not a trivial fix

Reply:
Fixed in 1e2e62c